apps/03TicTacToe.js

```diff
@@ -14,80 +14,176 @@ var board = [
 
 var playerTurn = 'X';
 
+function togglePlayer() {
+  playerTurn = (playerTurn === 'X') ? 'O' : 'X';
+}
+
+function clearBoard() {
+  board = [
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
+  ];
+}
+
 function printBoard() {
+  console.log(' ');
   console.log('   0  1  2');
   console.log('0 ' + board[0].join(' | '));
   console.log('  ---------');
   console.log('1 ' + board[1].join(' | '));
   console.log('  ---------');
   console.log('2 ' + board[2].join(' | '));
+  console.log(' ');
 }
 
 function horizontalWin() {
   // Your code here
+  return (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) || (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) || (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)
+}
+
+function horizontalRowWin(row) {
+  return (board[row][0] === playerTurn && board[row][1] === playerTurn && board[row][2] === playerTurn)
 }
 
 function verticalWin() {
   // Your code here
+  return (board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) || (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) || (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn)
 }
 
 function diagonalWin() {
   // Your code here
+  return (board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) || (board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn)
 }
 
 function checkForWin() {
+  if (horizontalWin() === true) {
```

Comment:
You're right, thanks man. I was testing out a cleaner way of testing for win and forgot to delete it.